.circleci/config.yml

```diff
@@ -121,6 +122,58 @@ jobs:
           destination: integration/geb
       - store_test_results:
           path: integration-tests/build/test-results
+  integration_tests_cypress:
```

Comment:
We may want to set the resource class depending on how this build executor runs. Worth keeping an eye on.